package.json

```diff
@@ -32,15 +32,15 @@
     ]
   },
   "dependencies": {
-    "d3-color": "^2.0.0",
+    "d3-color": "3.1.0",
     "d3-scale": "^3.2.3",
     "d3-scale-chromatic": "^2.0.0",
     "d3-shape": "^2.0.0",
-    "styled-components": "^4.3.2"
+    "react": "^18.2.0",
```

Comment:
Could you please keep react and react-dom as peer dependencies while upgrading the versions?